### DISCUSSION TITLE
Speed hack not working?

### DISCUSSION THREAD

**Discussion by TakeyaSaito** [julianxhokaxhiu/FFNx#425] [Q&A] [answered]
For the life of me i can't get speed hacks to work at all on FFVII, i have literally tried all the keys on my keyboard and nothing actually activates it? what is the hotkey meant to be? i can't find any documentation about it

**Comment by julianxhokaxhiu** [comment, marked as answer]
Hi, please consider looking at the documentation before opening an issue. Thanks!

https://github.com/julianxhokaxhiu/FFNx/blob/master/docs/shortcuts.md#speedhack
